src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -99,8 +140,8 @@
         :required-attributes (find-required-attributes library-schema)}))
    :render
    (fn [{:keys [props state this]}]
-     (let [{:keys [library-schema]} props
-           {:keys [page-num invalid-properties working-attributes published? required-attributes validation-error submit-error]} @state]
+     (let [{:keys [library-schema writer? curator?]} props
+           {:keys [page-num pages-seen invalid-properties working-attributes published? required-attributes validation-error submit-error]} @state]
```

Comment:
with your orch PR, is `curator` even used anymore? I worry that UI now enforces curator in some places that the API does not.